flash/core/integrations/icevision/transforms.py

```diff
@@ -118,16 +140,26 @@ def from_icevision_detection(record: "BaseRecord"):
             for bbox in detection.bboxes
         ]
 
-    if hasattr(detection, "masks"):
-        masks = detection.masks
-
-        if isinstance(masks, EncodedRLEs):
-            masks = masks.to_mask(record.height, record.width)
+    mask_array = (
+        getattr(detection, "mask_array", None) if _ICEVISION_GREATER_EQUAL_0_11_0 else getattr(detection, "masks", None)
+    )
+    if mask_array is not None:
+        if isinstance(mask_array, EncodedRLEs):
+            mask_array = mask_array.to_mask(record.height, record.width)
 
-        if isinstance(masks, MaskArray):
-            result["masks"] = masks.data
+        if isinstance(mask_array, MaskArray):
+            result["mask_array"] = mask_array.data
         else:
-            raise RuntimeError("Masks are expected to be a MaskArray or EncodedRLEs.")
+            raise RuntimeError("Mask arrays are expected to be a MaskArray or EncodedRLEs.")
+
+    masks = getattr(detection, "masks", None)
```

Comment:
```suggestion
    masks = getattr(detection, "masks")
```
I think None is default

Reply:
The default behaviour is to raise an attribute error if the attribute isn't there (which it may not be here if you are doing detection).